GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3088,6 +3105,16 @@ private string GetMonoVersion()
             return null;
         }
 
+        private bool DeleteSelectedFiles()
+        {
+            if (DiffFiles.Focused)
+            {
+                diffDeleteFileToolStripMenuItemClick(this, null);
```

Comment:
please don't call handlers directly, extract the share functionality into own method